README.md

```diff
@@ -87,8 +87,8 @@ Related projects
     [DefinitelyTyped](https://github.com/borisyankov/DefinitelyTyped)
     interfaces and produce OCaml interfaces.
 
-  - [Bucklescript](https://github.com/bloomberg/bucklescript):
-    Another compiler from OCaml to Javascript.
+  - [ReScript](https://github.com/rescript-lang/rescript-compiler):
+    Another compiler from OCaml to Javascript, featuring the [genType](https://github.com/reason-association/genType) ppx for generating TS / Flow types and runtime converters.
```

Comment:
It's a minor point, but it might be a good idea to fix it at the same time.
`Javascript` -> `JavaScript`

Reply:
Oh no, it's wrong in more places than I thought. I will open another PR.

Reply:
Feel free to edit as you like!

Reply:
I'm not a developer working at LexiFi, and I'm not a collaborator, so I don't have edit privileges for this PR. So I will deal with this with another PR!